templates/default/index.php

```diff
@@ -38,6 +38,8 @@
 $isDebug       = defined('AKEEBADEBUG') && AKEEBADEBUG;
 $themeColor    = TemplateHelper::getThemeColour();
 $importMap     = TemplateHelper::getImportMapAsJson();
+$view          = $this->getContainer()->input->getCmd('view', 'main');
```

Comment:
Change this to `$view = $this->getContainer()->input->getCmd('view', 'main') ?: 'main';` to always have a value. FWIW you can never have an empty view in AWF, the Dispatcher always fills it in. This is just an extra safeguard.